docs/dictionary/property/layerMode.lcdoc

```diff
@@ -58,6 +62,8 @@ object with a non-copy/blendSrcOver ink set - effective layerMode is set
 to 'dynamic'. 
 3. You have a scrolling <layerMode> on a non-group or an
 adorned group - effective layerMode is set to 'dynamic'.
+4. You have a container <layerMode> on a non-group or an
```

Comment:
Does effective layerMode also return static if you have a container layerMode group inside a non-container layerMode group?

Reply:
the test for effective layer mode (in MCControl::layer_computeattrs) doesn't check the effective layermode or control type of the parent:

```
else if (m_layer_mode_hint == kMCLayerModeHintContainer)
	{
		// A container layer must be unadorned, non-opaque and a group.
		if (gettype() == CT_GROUP && !t_is_opaque && t_is_unadorned)
			t_layer_mode = kMCLayerModeHintContainer;
		else
			t_layer_mode = kMCLayerModeHintStatic;
	}
```

Reply:
Okay cool :)